packages/core/src/utils/transformers/media/transformMedia.ts

```diff
@@ -39,7 +39,7 @@ export const transformMedia = (
       value = getOptimizedImageAsset(
         asset.fields.file as AssetFile,
         options.targetSize as string,
-        Number(options.quality),
+        options.quality,
```

Comment:
I'm seeing an inconsistency from the UI side.

The `cfImageOptions` variable starts with the quality value as `'100'`
* https://github.com/contentful/experience-builder/blob/b2117a703ae8b5ed603e27b481f4764b6e830a1c/packages/core/src/definitions/styles.ts#L197

In the UI, the `<ImageQualityInput>`updates the quality value using
```ts
onBlur={(newValue) => onChange(newValue)}
```

This makes it so Image components that haven't changed the quality setting will use `'100'` but Image components that do change the quality will have the format included in the value.

![Screenshot 2024-03-20 at 6 44 01 AM](https://github.com/contentful/experience-builder/assets/8539634/0359aaae-7b24-421a-ada8-ddbae3fe2437)

----

Do you think we should change the default quality value to `'100%'` on the SDK side so it has the format included in both cases?

Or should update the UI side of this to parse out the format when the quality input is changed?

Reply:
As a side note, I wanted to switch quality to numeric when merging the image option variables, but the `<NumericUnitInput>` handles the value as a string and so keeping it the same as before was the path of least resistance
* https://github.com/contentful/experience-builder/pull/482/files#diff-1659ec2be3f58ed052d1e69cf3e41c3ec8b712dde447b853d562e472ce762afcL241-L246

Reply:
It works with or without the format now, but ya, being consistent is good. Since the NumericUnitInput contains the format in its change events, its good to have the format included in the string as it goes into it as well. Typically we don't need to worry about converting these values to numbers, but we have to here for some logic with the quality. I'll set the default to 80% (since thats in another ticket) and see if it causes any problems.